pkg/policy/consumer.go

```diff
@@ -258,37 +246,36 @@ func (c *Consumable) AllowConsumerAndReverseLocked(cache *ConsumableCache, id Nu
 		}).Debug("Allowing reverse direction")
 		if _, ok := reverse.ReverseRules[c.ID]; !ok {
 			reverse.addToMaps(c.ID)
-			reverse.ReverseRules[c.ID] = NewConsumer(c.ID)
+			reverse.ReverseRules[c.ID] = true
 			return true
 		}
 	}
 	log.WithFields(logrus.Fields{
 		logfields.Identity + ".from": c.ID,
 		logfields.Identity + ".to":   id,
-	}).Warn("Allowed a consumer which can't be found in the reverse direction")
+	}).Warn("Allowed an ingress security identity which can't be found in the reverse direction")
```

Comment:
Suggestion:
`Allowed a security identity on ingress, but couldn't [find/resolve] the identity for the reverse direction`

Reply:
Done.